src/benchUtil.c

```diff
@@ -979,6 +982,7 @@ int64_t convertDatatypeToDefaultMax(uint8_t type) {
             ret = 254;
             break;
         case TSDB_DATA_TYPE_SMALLINT:
+        case TSDB_DATA_TYPE_GEOMETRY:
```

Comment:
geometry/varbinary data could not to set max

Reply:
like point(x,y)  max is limit x and y